modules/nodes-nodes-kernel-arguments.adoc

```diff
@@ -135,12 +135,12 @@ $ oc debug node/ip-10-0-141-105.ec2.internal
 Starting pod/ip-10-0-141-105ec2internal-debug ...
 To use host binaries, run `chroot /host`
 
-sh-4.3# cat /host/proc/cmdline
+sh-4.4# cat /host/proc/cmdline
```

Comment:
And this file should be reverted

Reply:
See https://github.com/openshift/openshift-docs/pull/21456/files#r414132734